Resources/Prototypes/Loadouts/Jobs/Security/security_officer.yml

```diff
@@ -2,18 +2,12 @@
 - type: loadoutEffectGroup
   id: SeniorOfficer
   effects:
-  - !type:JobRequirementLoadoutEffect
```

Comment:
If these are Wizden requirements then this needs to be commented out instead of being deleted, and needs Delta-V comments